shared/src/common/enums/validation/master/master-sign-up.validation-message.ts

```diff
@@ -1,6 +1,11 @@
 const MasterSignUpValidationMessage = {
   EMAIL_REQUIRE: 'Email is required',
   EMAIL_WRONG: 'Email is wrong',
+  NAME_REQUIRE: 'Name is required',
+  NAME_MIN_LENGTH: 'Name must have at least 4 characters',
```

Comment:
create enum `MasterSignUpValidationRule` and use here (for all statements)

```suggestion
  NAME_MIN_LENGTH: 'Name must have at least ${MasterSignUpValidationRule. NAME_MIN_LENGTH} characters',
```